feature-runner/steps/device.ts

```diff
@@ -37,14 +37,15 @@ const createDeviceForModel =
 			Type.Object({
 				model: Type.String(),
 				storageName: Type.String(),
+				account: Type.String(),
 			}),
 		)(
-			/^I have the fingerprint for a `(?<model>[^`]+)` device in `(?<storageName>[^`]+)`$/,
+			/^I have the fingerprint under `(?<account>[^`]+)` account for a `(?<model>[^`]+)` device in `(?<storageName>[^`]+)`$/,
```

Comment:
```suggestion
			/^I have the fingerprint for a `(?<model>[^`]+)` device(?<maybeAccount> in the `(?<account>[^`]+)` account) in `(?<storageName>[^`]+)`$/,
```

Make the account optional, and use the default if not needed during a test.